### ISSUE TITLE
Accessibility Improvements

### ISSUE THREAD

**Issue by bradledford** [sthomas1618/react-crane#2]
Select control needs to be able to be navigated (expanded/collapsed) somehow using the keyboard. This commonly is done through key bindings for Return or Down Arrow while the control has focus, then allowing up and down arrow key navigation within the list of values.

**Comment by sthomas1618** [comment]
Partially done by d1b1e34

Events done: Up, Down, Escape, Enter, Tab

Some more events still need to be handled, like PageUp, PageDown, Home, End, Backspace

**Comment by jaiswal12345** [comment]
I don't see any docs for its implementation.Is there any chance of adding that?

**Comment by sthomas1618** [comment]
@jaiswal12345  Yes! I hope to find time to improve docs and also modernize the repo soon 😄
